gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4.1-bin.zip
```

Comment:
Shouldn't this be "all" still?

Reply:
This is what it generated by the `Wrapper` task. It doesn't seem to want/need the `-all` artifact.

Reply:
Turns out that this is configurable.